server/views/pages/error.njk

```diff
@@ -9,11 +10,25 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
-      <h2 class="govuk-heading-m">{{ status }}</h2>
-      <p>{{ message }}</p>
+      <h1 class="govuk-heading-l">{{ heading }}</h1>
+      {% for item in message %}
+        <p>{{ item }}</p>
+      {% endfor %}
+
+      {% if showHomeButton %}
+        {{ govukButton({
```

Comment:
Could replace actually rendering the button here with a component we already have:
```
{% include "partials/backToStartButton.njk" %}
```

This has the advantage of the URL being `/back-to-start` which is a route that clears session and lands you on the home page. If making this change, would need to tweak the test as it will perhaps have different data selector and URL

Reply:
completely forgot about this partial/route, thanks!